drag_and_drop_v2/default_data.py

```diff
@@ -31,10 +31,6 @@
       "zone": "Zone 1",
       "backgroundImage": "",
       "id": 0,
-      "size": {
```

Comment:
@bradenmacdonald it looks like this is still needed - I can't edit fresh DnDs, because it crash with unhandled excpeption "Cannot read property 'width' of undefined" at `drag_and_drop.js:326`, which is invoked from `drag_and_drop.js:92`.